ecommerce/shopper_experience/payment_acquirer.rst

```diff
@@ -1,78 +1,70 @@
    method.
 
+Automatically generate invoices at order
+========================================
 
-Auto-validate invoices at order
-===============================
-
-When the order is confirmed you can also have an invoice automatically issued
-and paid. This fully-automated made for businesses that invoice 
+When the order is confirmed, you can also have an invoice automatically issued
+and paid. This fully-automated feature is designed for businesses that invoice 
 orders straight on.
 
-.. image:: media/payment_invoice.png
-    :align: center
+To do this, go to the *Invoicing* section of the *Settings* in the Website application,
+and select *Invoice what is ordered* and then select *Automatic Invoices*, which will
+generate an invoice automatically when the online payment is confirmed.
+
+.. image:: media/automatic-invoice.png
+   :align: center
+   :alt: example of automatic invoice
 
 .. note::
-   If you choose this mode you are requested to select a payment journal in order to record payments
-   in your books. This payment is automatically reconcilied with the invoice, marking it as paid.
-   Select your **bank account** if you get paid immediately on your bank account. If you don't you
-   can create a specific journal for the payment acquirer (type = Bank). That way, you can track
+   If you choose this mode, you are requested to select a payment journal, in order to record 
+   payments in your books. This payment is automatically reconciled with the invoice, marking it as
+   paid. Select your **bank account** if you get paid immediately on your bank account. If you don't 
+   you can create a specific journal for the payment acquirer (type = Bank). That way, you can track
    online payments in an intermediary account of your books until you get paid into your bank
    account (see :doc:`../../general/payment_acquirers/payment_acquirers`).
 
+Capture payment after the delivery
+==================================
+With this mode (which is only available with Authorize.net), the order is confirmed, 
+but the amount is kept on hold. Once the delivery is processed, you can capture the payment from
+Odoo.
 
-Capture the payment after the delivery
-======================================
-With this mode, the order is confirmed but the amount is kept on hold. 
-Once the delivery processed, you can capture the payment from Odoo.
-This mode is only available with Authorize.net.
-
-.. image:: media/payment_capture_mode.png
-    :align: center
-
-To capture the payment, open the transaction from the order.
-Then click *Capture Transaction*.
-
-.. image:: media/payment_capture.png
-    :align: center
-
-With other payment acquirers, you can manage the capture in
-their own interfaces, not from Odoo.
\ No newline at end of file
+To learn more about that, simply follow the procedure explained `here <https://www.odoo
+.com/documentation/user/14.0/general/payment_acquirers/authorize
+.html#capture-the-payment-after-the-delivery>`_.
```

Comment:
Never use external links to target an internal doc or a specific part of a doc. Also, try to add the text on a meaningful description of the link rather than on "here" or "click here".

Instead, you can use either a `:doc:` directive or an internal hyperlink target. (See [RST cheat sheet: hyperlinks](https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_cheat_sheet.html#hyperlinks))

In this case, since the purpose of the paragraph is to refer to other docs, we can replace it with a [seealso](https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_cheat_sheet.html#seealso) section.

```suggestion
.. seealso::
   - :doc:`../../general/payment_acquirers/payment_acquirers`
   - :ref:`Payment Acquirers: Place a hold on a card <payment_acquirers/capture_amount>`
   - :doc:`../../general/payment_acquirers/authorize`
```

![image](https://user-images.githubusercontent.com/46810843/116718403-36a9e280-a9da-11eb-9be5-2f9729c60a8b.png)